YoutubeDownloader/Views/Dialogs/AuthSetupView.xaml

```diff
@@ -53,11 +56,24 @@
                         TextAlignment="Center"
                         TextWrapping="Wrap">
                         <Run FontWeight="Light" Text="You are currently authenticated" />
-                        <LineBreak />
-                        <Hyperlink x:Name="LogoutHyperlink" Click="LogoutHyperlink_OnClick">
-                            Log out
-                        </Hyperlink>
                     </TextBlock>
+
+                    <Button Theme="{StaticResource MaterialFlatButton}"
+                            assists:ButtonAssist.HoverColor="Transparent"
+                            assists:ButtonAssist.ClickFeedbackColor="Transparent"
+                            Click="LogoutHyperlink_OnClick"
+                            HorizontalAlignment="Center"
+                            Cursor="Hand"
+                            Padding="0">
+                        <Button.Styles>
+                            <Style Selector="Button:pointerover TextBlock">
+                                <Setter Property="Foreground" Value="{DynamicResource SecondaryHueMidBrush}"/>
+                            </Style>
+                        </Button.Styles>
+                        <TextBlock Text="Log out" FontSize="18"
+                                   IsHitTestVisible="True" TextDecorations="Underline">
+                        </TextBlock>
+                    </Button>
```

Comment:
By the way, if it would simply the code to render this part as a regular button instead of a pseudo-hyperlink inside of a button, then I'm good with that too.

Reply:
yes, it would be much simpler, which one do you want?
![grafik](https://github.com/Tyrrrz/YoutubeDownloader/assets/33566379/249b2f14-f8a0-4695-a85d-265e6e6cf785)
![grafik](https://github.com/Tyrrrz/YoutubeDownloader/assets/33566379/a9affdeb-2895-4a03-b22d-6c190541e466)

Reply:
I think the last one in the bottom right makes the most sense